automathon/finite_automata/nfa.py

```diff
@@ -377,13 +377,11 @@ def __get_e_closure(
             for st in self.delta[q][""]:
                 if st not in visited:
                     visited.append(st)
-                    ans.extend(
-                        [
-                            k
-                            for k in self.__get_e_closure(st, visited)
-                            if k not in ans
-                        ]
-                    )
```

Comment:
I think the ruff formatter made some unintended changes, let me know if you want me to leave it like it was.